src/main/java/uk/gov/pay/directdebit/payments/services/sandbox/SandboxEventToPaymentStateMapper.java

```diff
@@ -1,17 +1,17 @@
 package uk.gov.pay.directdebit.payments.services.sandbox;
 
 import uk.gov.pay.directdebit.common.model.DirectDebitStateWithDetails;
+import uk.gov.pay.directdebit.events.model.GoCardlessPaymentAction;
 import uk.gov.pay.directdebit.events.model.SandboxEvent;
 import uk.gov.pay.directdebit.payments.model.PaymentState;
-import uk.gov.pay.directdebit.webhook.gocardless.services.handlers.GoCardlessPaymentHandler;
 
 import java.util.Map;
 import java.util.Optional;
 import java.util.Set;
 
 public class SandboxEventToPaymentStateMapper {
     private static final Map<String, PaymentState> SANDBOX_ACTION_TO_PAYMENT_STATE = Map.of(
-            GoCardlessPaymentHandler.GoCardlessPaymentAction.PAID_OUT.toString(), PaymentState.PAID_OUT
+            GoCardlessPaymentAction.PAID_OUT.toString(), PaymentState.PAID_OUT
```

Comment:
Something’s gone wrong here… we shouldn’t have GoCardless actions in a mapper used by sandbox!